test/libsolidity/semanticTests/revertStrings/function_entry_checks_v1.sol

```diff
@@ -4,6 +4,7 @@ contract C {
 }
 // ====
 // EVMVersion: >=byzantium
+// ABIEncoderV1Only: true
```

Comment:
This test already had `compileViaYul: false` and the pragma, but adding `ABIEncoderV1Only` makes it explicit (and searchable).